manifests/rule.pp

```diff
@@ -80,6 +84,22 @@
     fail('Exactly one of "action" or the deprecated "policy" param is required.')
   }
 
+  if $outerface and !($chain in ['FORWARD', 'OUTPUT', 'POSTROUTING']) {
+    fail('Outgoing interface can only be set in the "FORWARD", "OUTPUT" and "POSTROUTING" chains.')
+  } elsif $outerface {
+    $outerface_real = " outerface ${outerface}"
+  } else {
+    $outerface_real = ''
+  }
+
+  if $to_source and $action_temp != 'SNAT' {
```

Comment:
`SNAT` is only valid in the `POSTROUTING` chain of the `nat` table, you should make sure to enforce this here.

Reply:
I don't see how that's different than before this PR, but I added a check for that anyway.